src/cogs/when_next.py

```diff
@@ -27,48 +27,44 @@ def __init__(self, bot):
         self.logging = logging.getLogger(__name__)
 
     @commands.command(name="whennext")
-    async def entry(self, context, cmd: str):
+    async def entry(self, context, days: str = "31", weekdays: str = "567", start_offset: str = "0"):
         self.logging.info("whennext called")
-        arg_dict = self._parse_command(cmd)
-        from_date = datetime.date.today()
-        if FROM_KEY in arg_dict:
-            from_date = self._parse_date(context, arg_dict[FROM_KEY])
-            if from_date is False:
-                return
-        until_date = from_date + datetime.timedelta(weeks=2)
-        if UNTIL_KEY in arg_dict:
-            until_date = self._parse_date(context, arg_dict[UNTIL_KEY])
-            if until_date is False:
-                return
-        weekend_days = []
+        days_int = self._parse_int(context, "days", days)
+        if days_int is False:
```

Comment:
I don't know if you used `is` over `==` deliberately, but despite being a syntax warning, this saves you if `days_int == 0` lol

Reply:
I think I just googled which way you should check falsiness in python and used that